packages/shared/src/hooks/profile/useProfile.ts

```diff
@@ -13,8 +12,7 @@ export function useProfile(initialUser?: PublicProfile): PublicProfile {
     generateQueryKey(RequestKey.Profile, initialUser),
     () => getProfile(initialUser?.id),
     {
-      ...disabledRefetch,
```

Comment:
We don't really need to disable refetch because we now have stale time (it wont refetch until that is elapsed or we use `invalidateQueries` (like during profile update)